shibata added the new strings in the Japanese version to the English version. (#83)
fujitake/vantiq-related: vantiq-cloud-infra-operations/terraform_azure/readme_en.md

  ```
  - Since we will be using Bs family, ESv3 family, and FSv2 family, increase from the default value of 10. (30~50 is enough)

- Secure VNET IP range. The range of subnets should be greater than /22.  In case of Production configuration, the cluster will have to occupy 30 IPs per node so 11 nodes cannot be accommodated in /24.

> - Ensure that the build environment can communicate to the Internet in order to download the information necessary for installation. Especially when a closed network is configured with a firewall.

Reviewer: ```suggestion
- Ensure that the build environment can reach the Internet in order to download the necessary files for installation. Especially if you are building one in a closed network with the firewall.
```

>   - [Azure Global required FQDN / application rules](https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic#azure-global-required-fqdn--application-rules) - If communication with containers, repositories, etc. is not possible, AKS construction will fail.

Reviewer: ```suggestion
  - [Azure Global required FQDN / application rules](https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic#azure-global-required-fqdn--application-rules) - AKS construction will fail if it cannot reach the container repository.
```

>   - The Vantiq Bastion node should be able to communicate with the software repositories required for the Vantiq installation. Since it is difficult to register all of them in the whitelist, it is preferable to allow them uniformly.

Reviewer: ```suggestion
  - The Vantiq Bastion node should be able to reach the software repositories required for the Vantiq installation. As it is difficult to whitelist all target repositories, it is recommended to allow all outbound connection.
```
